src/js/api/core-shim.js

```diff
@@ -244,18 +244,22 @@ Object.assign(CoreShim.prototype, {
 function setupError(core, error) {
     resolved.then(() => {
         let errorEvent = error;
+        const model = core._model || core.modelShim;
+
         if (!isValidNumber(error.code)) {
             // Transform any unhandled error into a PlayerError so emitted events adhere to a uniform structure
-            errorEvent = new PlayerError(error.message, SETUP_ERROR_UNKNOWN, error);
+            errorEvent = new PlayerError(MSG_TECHNICAL_ERROR, SETUP_ERROR_UNKNOWN, error);
         }
+        // The message may have already been created (eg. multiple players on a page where a plugin fails to load)
+        errorEvent.message = errorEvent.message || model.get('localization').errors[errorEvent.key];
+        errorEvent.key = null;
```

Comment:
Let's no output undocumented properties with null values in our API events. @egreaves @johnBartos better to deop on error than degrade the developer experience.